src/Compiler/Checking/PostInferenceChecks.fs

```diff
@@ -302,12 +302,24 @@ let BindVal cenv env (v: Val) =
     //printfn "binding %s..." v.DisplayName
     let alreadyDone = cenv.boundVals.ContainsKey v.Stamp
     cenv.boundVals[v.Stamp] <- 1
+    
+    let topLevelPublicLetBindingNotInSignatureFile () =
```

Comment:
That name implies it'll return `true` when there's no signature file present at all, but it's not the case. Maybe something like `hiddenBySignatureFile`?